hpct-ldap-server/src/charm.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical
+# See LICENSE file for licensing details.
+"""OPENLDAP SERVER OPERATOR.
+
+This operator provides directory services for all HPC-related operations as a service.
+
+Utilizes a peer relation for failover server units.
+Utilizes a provides relation for clients to connect to the server.
+
+charmcraft -v pack
+juju deploy ./openldap-server-operator_ubuntu-22.04-amd64.charm -n <#-of-failover-units>
+
+For more info see:
+https://hpc4can.ddns.net/xwiki/bin/view/Users/dvgomez/HPC%20LDAP%20Server%20and%20Client%20Operators/#
```

Comment:
Remove in favor of `README.md`. Also, we do not want to refer to the xwiki which is private.

Reply:
Updated as suggested